src/torchmetrics/detection/panoptic_quality.py

```diff
@@ -48,11 +48,17 @@ class PanopticQuality(Metric):
             stuffs:
                 Set of ``category_id`` for uncountable stuffs.
             allow_unknown_preds_category:
-                Bool indication if unknown categories in preds is allowed
+                Boolean flag to specify if unknown categories in the predictions are to be ignored in the metric
+                computation or raise an exception when found.
+
+        TODO(marco): where to say that points in target that are not things or stuff
```

Comment:
This is something we should address in this PR, since it's not an obvious behaviour and just about every user needs to know how to define "ignore_index" kind of logic

Reply:
[178e715](https://github.com/Lightning-AI/metrics/pull/1527/commits/178e7158cfa9a497d2ac2621075c4bdce5cde853)